packages/client/components/JiraFieldMenu.tsx

```diff
@@ -23,7 +23,7 @@ const JiraFieldMenu = (props: Props) => {
   const {meetingId, dimensionRef, serviceField, task} = stage
   if (task?.integration?.__typename !== 'JiraIssue') return null
   const {integration} = task
-  const {cloudId, projectKey, possibleEstimationFieldNames} = integration
```

Comment:
+1 this isn't the line of code where the problem is happening, but I wanted to note it down:

If a project doesn't have a Story Points field, the menu should probably not show the divider or the padding at the top:
![jira-menu](https://user-images.githubusercontent.com/39854876/195387433-991884ed-7e7e-4811-8c08-aac3fe594264.png)